packages/opentelemetry-node/lib/instrumentations.js

```diff
@@ -179,22 +215,48 @@ const INSTRUMENTATIONS = {
 function getInstrumentations(opts = {}) {
     /** @type {Array<Instrumentation>} */
     const instrumentations = [];
+    const enabledFromEnv = getInstrumentationsFromEnv(
+        'OTEL_NODE_ENABLED_INSTRUMENTATIONS'
+    );
+    const disabledFromEnv = getInstrumentationsFromEnv(
+        'OTEL_NODE_DISABLED_INSTRUMENTATIONS'
+    );
 
     Object.keys(INSTRUMENTATIONS).forEach((name) => {
+        // Skip if the env configuration says so
+        if (enabledFromEnv && !enabledFromEnv.includes(name)) {
+            return;
+        }
+        if (disabledFromEnv && disabledFromEnv.includes(name)) {
+            // Do not skip if it was in the enable list
+            if (!enabledFromEnv) {
```

Comment:
I think this is wrong. If an instrumentation name is listed in both OTEL_NODE_ENABLED_INSTRUMENTATIONS and OTEL_NODE_DISABLED_INSTRUMENTATIONS, then it should be *disabled*. I.e. disabling wins. That is what auto-instr-node is doing:

https://github.com/open-telemetry/opentelemetry-js-contrib/blob/60a99c98fd3a5594c7c2234184f06166b375e707/metapackages/auto-instrumentations-node/test/utils.test.ts#L117-L134